### ISSUE TITLE
Error in logs

### ISSUE THREAD

**Issue by jes1417** [gadgetchnnel/entities_calendar#4]
```
Log Details (ERROR)

Logger: homeassistant.helpers.entity
Source: custom_components/entities_calendar/calendar.py:287
First occurred: 12:29:38 AM (1 occurrences)
Last logged: 12:29:38 AM

Update for calendar.pet_feedings failsTraceback (most recent call last): File "/usr/src/homeassistant/homeassistant/helpers/entity.py", line 263, in async_update_ha_state await self.async_device_update() File "/usr/src/homeassistant/homeassistant/helpers/entity.py", line 456, in async_device_update await self.hass.async_add_executor_job( File "/usr/local/lib/python3.8/concurrent/futures/thread.py", line 57, in run result = self.fn(*self.args, **self.kwargs) File "/config/custom_components/entities_calendar/calendar.py", line 173, in update self.data.update() File "/usr/src/homeassistant/homeassistant/util/__init__.py", line 239, in wrapper result = method(*args, **kwargs) File "/config/custom_components/entities_calendar/calendar.py", line 287, in update self.event = events[0] IndexError: list index out of range

```

I have been seeing this error in the last few updates and the entities are not displaying I'm my calendar any more. Do you have any insight on what might be causing it?

**Comment by gadgetchnnel** [comment]
It appears that there was an issue where, if none of the entities associated with a calendar are available when it updates, it would cause this error (may be due to changes in Home Assistant to make startup faster).
I've just released version 0.0.6, which will should fix this. Can you please update and give it a try?

**Comment by jes1417** [comment]
The update fixed the issue, integration is working again. Thanks for the quick fix.
